### ISSUE TITLE
Homepage Broken

### ISSUE THREAD

**Issue by spectravp** [zendframework/zf-web#157]
Broken rendering in latest version of Safari

![screen shot 2015-01-03 at 4 04 36 pm](https://cloud.githubusercontent.com/assets/272688/5603748/74fc24ca-9362-11e4-8e20-65b8d99dcb53.png)

**Comment by froschdesign** [comment]
@spectravp 
You are right. Bad HTML.

Better:
```html
<div id="background">
    <div class="row">
        <div class="twelve columns">
            <h1 class="text-center">Maximize Development Productivity</h1>
        </div>
    </div>
    <div class="row dev-tools">
        <div class="four columns">
            <a href="https://apigility.org">
                <img src="images/ag-hero.png" alt="Apigility" width="200">
            </a>
            <p class="caption">
                The world's easiest way to create high-quality APIs.
            </p>
            <a href="https://apigility.org">
                <img src="images/Learn-more-button-166x37px.png" alt="Learn more">
            </a>
        </div>
        <div class="four columns">
            <a href="http://www.zend.com/en/products/server/z-ray">
                <img src="images/Z-ray-new-icon-146x90px.png" alt="Z-Ray">
            </a>
            <p class="caption">
                Write Better Code Faster.
            </p>
            <a href="http://www.zend.com/en/products/server/z-ray">
                <img src="images/Learn-more-button-166x37px.png" alt="Learn more">
            </a>
        </div>
        <div class="four columns">
            <a href="http://www.zend.com/en/products/studio">
                <img src="images/zend-studio-logo-346x54px.png" alt="Zend Studio">
            </a>
            <p class="caption">
                The PHP IDE for Smarter Development.
            </p>
            <a href="http://www.zend.com/en/products/studio">
                <img src="images/Learn-more-button-166x37px.png" alt="Learn more">
            </a>
        </div>
    </div>
</div>
```
We should also replace the image buttons.
I will create a PR.

Thanks for reporting!

**Comment by froschdesign** [comment]
> Bad HTML.

…and `flex-box`. :wink:

**Comment by froschdesign** [comment]
Prefix for Safari is missing. See: http://caniuse.com/#feat=flexbox

But I prefer the Foundation grid system to support mobile devices and reducing the CSS code.

**Comment by weierophinney** [comment]
I have a mixin available for making the flexbox stuff work cross-platform. I'd actually prefer to use it because... I was unable to find a working solution using Foundation. I know you were able to do so, but I spent a couple of hours trying different Foundation grid combinations before I decided to just fallback to flexbox... and I had a working solution in minutes once I did. The HTML was far less verbose, too.

I'll get a PR in place with the mixin for @froschdesign and @spectravp to test shortly.
